apps/omg_child_chain/lib/omg_child_chain/api/blocks_cache/storage.ex

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.ChildChain.API.BlocksCache.Storage do
+  @moduledoc """
+  Logic of the service to serve freshest blocks quickly.
+  """
+  alias OMG.Block
+  alias OMG.DB
+
+  @doc """
```

Comment:
consider to move this doc under `API.BlocksCache`.

I was not aware why extra Genserver layer there until the explanation of forcing request to sit in queue in here.